openupgrade_scripts/scripts/l10n_es_edi_verifactu/18.0.1.0/upgrade_analysis.txt

```diff
@@ -1,44 +1,17 @@
 ---Models in module 'l10n_es_edi_verifactu'---
-new model l10n_es_edi_verifactu.document
+obsolete model l10n_es_edi_verifactu.certificate
```

Comment:
what happened here? which odoo commit affected?

Reply:
I don't know. It feels very weird...

Reply:
`verifactu` was merged in v17 (https://github.com/odoo/odoo/pull/197635) on July 22nd, which made https://github.com/OCA/OpenUpgrade/pull/5266 (on August 4th). Later, `verifactu` was merged in v18 (https://github.com/odoo/odoo/pull/219953) on July 30th, which made this PR (on August 5th).

Then the PR is correct, but it seems the analysis github action takes older Odoo dates. @hbrunn could you check if everything is fine regarding dates?

Reply:
the analysis takes the [latest archive of the OCB version](https://github.com/OCA/OpenUpgrade/blob/17.0/.github/workflows/generate-analysis.yml#L92), which indeed lags behind a bit. Maybe better switch this to vanilla Odoo?

Reply:
Yes, better to use vanilla Odoo.

Reply:
@hbrunn how can we change it to vanilla Odoo?